OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/PDFViewerTopComponent.java

```diff
@@ -280,7 +286,16 @@ public void componentClosed() {
     
     private void refreshDisplayPane() {
         JPanel pdfImagePanel = pdfDisplay.drawPreviewOnJPanel();
-       
+        if(PDFDisplay.isDisableNextFlag()){
+            jButton2.setEnabled(false);
+        } else{
+            jButton2.setEnabled(true);
+        }
+        if(PDFDisplay.isDisablePrevFlag()){
+            jButton1.setEnabled(false);
+        } else{
+            jButton1.setEnabled(true);
+        }
```

Comment:
We set buttons enabled/disabled in two places in this class and both places use different condition to do so. This is not good.